exporter/debugexporter/factory.go

```diff
@@ -28,13 +30,14 @@ const (
 )
 
 // NewFactory creates a factory for Debug exporter
-func NewFactory() exporter.Factory {
-	return exporter.NewFactory(
+func NewFactory() exporterprofiles.Factory {
```

Comment:
Same comment you MUST return `exporter.Factory` here.

Reply:
Thanks again @bogdandrutu! Fixed ✅